tools/networkmonitor/networkmonitor.nim

```diff
@@ -81,13 +82,19 @@ proc setConnectedPeersMetrics(discoveredNodes: seq[Node],
     allPeers[peerId].ip = ip
 
     # get more info the peers from its ip address
-    let location = await ipToLocation(ip, client)
-    if not location.isOk():
+    var location: NodeLocation
+    try:
+      # IP-API endpoints are now limited to 45 HTTP requests per minute
+      # TODO: As network grows, find a better way to now block the whole app
```

Comment:
```suggestion
      # TODO: As network grows, find a better way to not block the whole app
```
:) Just in case someone in future believes they should _now_ really block everything.